ui/v2.5/src/components/Scenes/styles.scss

```
@@ -901,3 +901,18 @@ input[type="range"].blue-slider {
     word-break: break-all;
   }
 }
+
+.scene-list .filtered-list-toolbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+.scene-list.hide-sidebar .sidebar-toggle-button {
+  transition-delay: 0.1s;
+  transition-duration: 0;
+  transition-property: opacity;
+}
+
+.scene-list:not(.hide-sidebar) .sidebar-toggle-button {
+  opacity: 0;
```

I think I understand the reason you set the opacity here rather than setting `display: none`. However, the issue with just setting opacity to 0 is that the invisible button remains engageable if a user's mouse happens to stumble onto its location on the page. We can avoid this by also setting `pointer-events: none;`
```suggestion
  opacity: 0;
  pointer-events: none;
```